### PR TITLE
Django 1.10 compability

### PR DESCRIPTION
**Pull request by john-nguyen09**
Added django 1.10 support, according to https://docs.djangoproject.com/en/1.10/topics/http/middleware/#upgrading-middleware

### CODE REVIEW

File: htmlmin/middleware.py [hunk 1]
```diff
@@ -9,37 +9,59 @@
...
+		return req_ok and resp_ok
+
+
+	def process_response(self, request, response):
+		minify = getattr(settings, "HTML_MINIFY", not settings.DEBUG)
+		keep_comments = getattr(settings, 'KEEP_COMMENTS_ON_MINIFYING', False)
+		parser = getattr(settings, 'HTML_MIN_PARSER', 'html5lib')
+		if minify and self.can_minify_response(request, response):
+			response.content = html_minify(response.content,
+											ignore_comments=not keep_comments,
```
Comment (andrewsmedina) [inline htmlmin/middleware.py]: can you fix the indentation here?

**Comment by bartmika** [comment]
This ticket is also associated with ticket #102.

**Comment by rodolfomartinez** [comment]
+1

**Comment by andrewsmedina** [comment]
@john-nguyen09 thank you!

**Pull request merged**